ci/2.7.pylintrc

```diff
@@ -1,113 +1,38 @@
-# unnecessary-lambda<W0108>: Used when the body of a lambda expression is a function call on the same argument list as the lambda itself
-# unnecessary-pass<W0107>: (hi-pri) Used when a "pass" statement that can be avoided is encountered.
-# unneeded-not<C0113>: (needs review) Used when a boolean expression contains an unneeded negation.
-# unused-argument<W0613>: (hi-pri) Used when a function or method argument is not used.
-# unused-import<W0611>: (hi-pri) Used when an imported module or variable is not used.
-# unused-variable<W0612>: (hi-pri) Used when a variable is defined but not used.
-# useless-else-on-loop<W0120>: (hi-pri) Loops should only have an else clause if they can exit early with a break statement, otherwise the statements under else should be on the same scope as the loop itself.
-# useless-super-delegation<W0235>: Used whenever we can detect that an overridden method is useless, relying on super() delegation to do the same thing as another method from the MRO.
-
-# The source code suppresses the following error codes specifically for python2.7:
-
-# no-init<W0232>: (needs review) Used when a class has no __init__ method, neither its parent classes.
-# undefined-loop-variable<W0631>: (needs review) Used when an loop variable (i.e. defined by a for loop or a list comprehension or a generator expression) is used outside the loop.
-# unsubscriptable-object<E1136>: (hi-pri) Emitted when a subscripted value doesn’t support subscription(i.e. doesn’t define __getitem__ method)
-
-# The source code suppresses the following error codes specifically for python3.4:
-
-# consider-using-in<R1714>: To check if a variable is equal to one of many values,combine the values into a tuple and check if the variable is contained "in" it instead of checking for equality against each of the values.
-# duplicate-string-formatting-argument<W1308>: (hi-pri) Used when we detect that a string formatting is repeating an argument instead of using named string arguments
-# implicit-str-concat-in-sequence<W1403>: (hi-pri) String literals are implicitly concatenated in a literal iterable definition : maybe a comma is missing ?
-# no-else-raise<R1720>: (hi-pri) Used in order to highlight an unnecessary block of code following an if containing a raise statement. 
-# simplifiable-if-expression<R1719>: Used when an if expression can be replaced with 'bool(test)'.
-# subprocess-popen-preexec-fn<W1509>: (hi-pri) The preexec_fn parameter is not safe to use in the presence of threads in your application. The child process could deadlock before exec is called.
-# useless-return<R1711>: (hi-pri) Emitted when a single "return" or "return None" statement is found at the end of function or method definition.
-
-
 [MESSAGES CONTROL]
 
-disable=bad-continuation, # Buggy, **REMOVED in pylint-2.6.0**
+disable=C, # (C) convention, for programming standard violation
```

Comment:
This replaces the specific, removed warnings below, right?

Reply:
It removes all the Convenition warnings:
>$ pylint --list-msgs | grep -E 'C[[:digit:]]{4}'
:blacklisted-name (C0102): *Black listed name "%s"*
:invalid-name (C0103): *%s name "%s" doesn't conform to %s*
:empty-docstring (C0112): *Empty %s docstring*
:unneeded-not (C0113): *Consider changing "%s" to "%s"*
:missing-module-docstring (C0114): *Missing module docstring*
:missing-class-docstring (C0115): *Missing class docstring*
:missing-function-docstring (C0116): *Missing function or method docstring*
:singleton-comparison (C0121): *Comparison to %s should be %s*
:misplaced-comparison-constant (C0122): *Comparison should be %s*
:unidiomatic-typecheck (C0123): *Using type() instead of isinstance() for a typecheck.*
:non-ascii-name (C0144): *%s name "%s" contains a non-ASCII unicode character*
:consider-using-enumerate (C0200): *Consider using enumerate instead of iterating with range and len*
:consider-iterating-dictionary (C0201): *Consider iterating the dictionary directly instead of calling .keys()*
:bad-classmethod-argument (C0202): *Class method %s should have %s as first argument*
:bad-mcs-method-argument (C0203): *Metaclass method %s should have %s as first argument*
:bad-mcs-classmethod-argument (C0204): *Metaclass class method %s should have %s as first argument*
:single-string-used-for-slots (C0205): *Class __slots__ should be a non-string iterable*
:line-too-long (C0301): *Line too long (%s/%s)*
:too-many-lines (C0302): *Too many lines in module (%s/%s)*
:trailing-whitespace (C0303): *Trailing whitespace*
:missing-final-newline (C0304): *Final newline missing*
:trailing-newlines (C0305): *Trailing newlines*
:multiple-statements (C0321): *More than one statement on a single line*
:superfluous-parens (C0325): *Unnecessary parens after %r keyword*
:mixed-line-endings (C0327): *Mixed line endings LF and CRLF*
:unexpected-line-ending-format (C0328): *Unexpected line ending format. There is '%s' while it should be '%s'.*
:wrong-spelling-in-comment (C0401): *Wrong spelling of a word '%s' in a comment:*
:wrong-spelling-in-docstring (C0402): *Wrong spelling of a word '%s' in a docstring:*
:invalid-characters-in-docstring (C0403): *Invalid characters %r in a docstring*
:multiple-imports (C0410): *Multiple imports on one line (%s)*
:wrong-import-order (C0411): *%s should be placed before %s*
:ungrouped-imports (C0412): *Imports from package %s are not grouped*
:wrong-import-position (C0413): *Import "%s" should be placed at the top of the module*
:useless-import-alias (C0414): *Import alias does not rename original package*
:import-outside-toplevel (C0415): *Import outside toplevel (%s)*
:len-as-condition (C1801): *Do not use `len(SEQUENCE)` without comparison to determine if a sequence is empty*

Reply:
Also, does this grouping include any warnings besides the ones removed below (and, of course, the warnings removed in this PR)?

Reply:
(of all the C warnings, the agent was hitting those:
>$ function pylint-rules {
>     pylint --list-msgs | sed -E '/^:([[:alpha:]-]{2,}).*\(C[[:digit:]]{4}\).*/!d;s//\1/' | sort -u
> }
$ function pylint-agent {
>     find azurelinuxagent tests -name '*.py' -exec sed -E '/^.*# pylint:\s?disable=([[:alpha:]-]{2,}).*$/!d;s//\1/' {} \; | sort -u
> }
$ 
$ comm -12 <(pylint-rules) <(pylint-agent)
blacklisted-name
invalid-name
len-as-condition
too-many-lines
ungrouped-imports
unidiomatic-typecheck

)